stdlib.ispc

```diff
@@ -2199,6 +2283,17 @@ static inline void *atomic_compare_exchange_local(void **ptr, void *oldval, void
 ///////////////////////////////////////////////////////////////////////////
 // Transcendentals (float precision)
 
+__declspec(safe) static inline float16 sqrt(float16 v) {
+    if (__math_lib == __math_lib_svml) {
+        const int16 iflt_neg_max = 0xff80; // -infinity
+        return halfbits(iflt_neg_max);     // FIXME: return __svml_sqrtf(v);
```

Comment:
How should we address `sqrt` for `__math_lib_svml`?

Reply:
We need to add SVML function for half types and check what platforms are supported. I'll have a look at it later today.